packages/adapter-firebase/package.json

```diff
@@ -12,35 +12,43 @@
     "Nico Domino <yo@ndo.dev>",
     "Alex Meuer <github@alexmeuer.com>"
   ],
-  "main": "dist/index.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js"
+    }
+  },
   "files": [
-    "dist",
-    "index.d.ts"
+    "*.js",
```

Comment:
This will include the config files like `jest.config.js` so I think it's best to be specific about what files
```suggestion
    "index.js",
```
Could you add the readme.md file here as well? 🙌

Reply:
:+1: FWIW, the readme is always included: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files:~:text=Certain%20files%20are%20always%20included

Tweaked a bit, `npm pack` now shows:
![image](https://user-images.githubusercontent.com/18369201/216820061-bb2148aa-13b8-4648-9112-553e8121c67c.png)